src/i_sndfile.c

```diff
@@ -483,6 +483,39 @@ static boolean OpenFile(sndfile_t *file, void *data, sf_count_t size)
     return true;
 }
 
+static void FadeInMono16(short *data, ALsizei size, ALsizei freq)
+{
+    const int fadelen = freq / 1000; // 1 ms
+    int i;
+
+    if (!data[0] || size / sizeof(short) < fadelen)
+    {
+        return;
+    }
+
+    for (i = 0; i < fadelen; i++)
+    {
+        data[i] = data[i] * i / fadelen;
+    }
+}
+
+static void FadeInMonoFloat32(float *data, ALsizei size, ALsizei freq)
+{
+    const int fadelen = freq / 1000; // 1 ms
+    int i;
+
+    if ((data[0] < 0.000001 && data[0] > -0.000001) ||
```

Comment:
Does Woof still support C89? Otherwise I would use `fabs()` or `fabsf()` here.

Reply:
We switched to C99 a while back. We can also use `int16_t` instead of `short`.